.clang-format

```diff
@@ -0,0 +1,43 @@
+# Modified from https://github.com/ament/ament_lint/blob/master/ament_clang_format/ament_clang_format/configuration/.clang-format
```

Comment:
You can add these config files by `sync-files` workflow.

Reply:
To make the review easier, it might be a good idea to run `sync-files` once and merge that first, then rebase this PR.

Reply:
Ah, I understand. Thank you for your instruction🙏 I will try it next time.

Reply:
How about merging `sync-files` first like this?
https://github.com/autowarefoundation/autoware.universe/pull/311

Reply:
I created https://github.com/tier4/ros2-project-template/pull/2. 🙏